model-server/src/test/kotlin/org/modelix/model/server/ModelClientV2Test.kt

```diff
@@ -154,4 +155,27 @@ class ModelClientV2Test {
 
         assertEquals("localhost", modelClient.getUserId())
     }
+
+    @Test
+    fun `newly created repository can be removed`() = runTest {
+        val url = "http://localhost/v2"
+        val client = ModelClientV2.builder().url(url).client(client).build().also { it.init() }
+        val repositoryId = RepositoryId(UUID.randomUUID().toString())
+        client.initRepository(repositoryId)
+
+        val success = client.deleteRepository(repositoryId)
+
+        assertEquals(true, success)
```

Comment:
I would also check if it's removed from the result of `client.listRepositories()`.

Reply:
Fixed in [f7bd1a0](https://github.com/modelix/modelix.core/pull/459/commits/f7bd1a0bf6390602ee5f7759a2c35fbd2f4e4625), see #459.